src/Storefront/Resources/views/storefront/component/listing/limit.html.twig

```diff
@@ -0,0 +1,15 @@
+{# @sw-package framework #}
```

Comment:
what about some blocks?

Reply:
This was inspired by https://github.com/shopware/shopware/blob/trunk/src/Storefront/Resources/views/storefront/component/sorting.html.twig which also doesn't have blocks 😄

Reply:
I mean, fewer blocks means less maintenance for us :man_shrugging: :grin: